### PR TITLE
Allow for filtering on 'taxonomies' on the overview pages.

### PR DESCRIPTION
**Pull request by bobdenotter**
It even works with paging, and it 'sticks' when using 'save and return to overview'. ^_^ 

See screenshot: 

screenshot_2015-03-29_13_07_13

### CODE REVIEW

**Comment by GawainLynch** [comment]
```
Key "taxonomy" for array with keys "name, singular_name, fields, default_status, 
show_on_dashboard, searchable, viewless, slug, singular_slug, show_in_menu, 
sort, groups" does not exist in "overview/_panel-actions_overview.twig" at line 24
```

**Comment by Pinpickle** [comment]
:tada:

File: app/view/twig/overview/_panel-actions_overview.twig [hunk 1]
```diff
@@ -14,14 +14,44 @@
...
-        <input type="text" class="form-control" value="{{ request('filter', '', true) }}" name="filter" style="width: 110px;" placeholder="{{ __('Filtering …') }}">
-        <button type="submit" class="btn btn-tertiary">{{ __('Filter') }}</button>
+    <p style="margin-top: 15px;"><strong>{{ __("Filter") }}</strong></p>
+
+    <form class="form-inline">
+
+        {% set taxonomyfilter = false %}
+
+        <div class="form-group">
+        {% for taxonomy in context.contenttype.taxonomy %}
```
Comment (GawainLynch) [inline app/view/twig/overview/_panel-actions_overview.twig]: Changing to this passes acceptance tests:
```twig
{% for taxonomy in context.contenttype.taxonomy|default([]])) %}
```

**Comment by GawainLynch** [comment]
It just fell in there, all on it's own!

**Comment by bobdenotter** [comment]
Ha, was I taking too long? ;-)

**Comment by GawainLynch** [comment]
I needed a sanity break, and figured you were having a busy-as-hell type of day.

**Pull request merged**